packages/ecommerce-app-base/src/AppConfig/AppConfig.spec.tsx

```diff
@@ -5,53 +5,7 @@ import { ConfigAppSDK } from '@contentful/app-sdk';
 
 import AppConfig from './AppConfig';
 import { definitions } from './parameters.spec';
-
```

Comment:
Copied/pasted this code out of `AppConfig` into `__mocks__` dir so that other test files can re-use this functionality.